README.md

```diff
@@ -128,15 +165,36 @@ OakMap<K,V> oak = builder.build();
 You are welcome to take a look on the OakMap's [full API](https://github.com/yahoo/Oak/wiki/Full-API).
 OakMap's API implements the ConcurrentNavigableMap interface. For improved performance, it offers additional non-standard zero-copy API methods that are discussed below.
 
-### OakBuffers
-OakMap provides two types of memory buffers: *OakRBuffer* (read-only) and *OakWBuffer* (read and write). These buffers support a similar API for read-only Java ByteBuffers and writable Java ByteBuffers, respectively.
-
-OakMap buffers allow the user direct access to the underlying serialized key-value pairs, without needing to worry about  concurrent accesses and memory management. This access reduces unnecessary copies and deserialization of the underlying mappings.
-Note, however, that since OakMap's get method avoids copying the value and instead returns access to the same underlying memory buffer that compute operations update in-place, the reader may encounter different values -- and even value deletions -- when accessing the buffer returned from get multiple times. This is of course normal behavior for a concurrent map that avoids copying.
-
-An OakRBuffer can represent either a key or a value. The OakRBuffer's user can use the standard interface of a *read-only* ByteBuffer, for example, `int getInt(int index)`, `char getChar(int index)`, `limit()`, etc. Note that ConcurrentModificationException can be thrown as a result of any OakRBuffer method in case the mapping is concurrently deleted.
-
-For backward compatibility with applications that are already based on the use of ByteBuffers, Oak Buffers provide the transform method that atomically applies a transformation to a *read-only* instance of the underlying ByteBuffer. For a more comprehensive code example please refer to the [usage](#usage) section. 
+### Oak Buffers
+Oak use dedicated buffer objects to access off-heap memory. 
+These buffers cannot be instantiated by the user and are always supplied to the user by Oak.
```

Comment:
What about minKey? Didn't you wanted it to be OakReadBuffer? Will we serialize from K ourselves?

Reply:
In the current implementation, the user supplies a min key as K.
We use the user's serialize function to convert it to a buffer.
Nothing changes there. Just that serialize will not be called with OakWriteBuffer.
Then we disable the write access to the buffer to prevent the user from changing it.